src/prediction_service_utils.hpp

```diff
@@ -27,6 +27,7 @@
 #include "shape.hpp"
 
 namespace ovms {
+class ModelInstanceUnloadGuard;
```

Comment:
Why do we need it?

Reply:
Removed dependency on modelinstance.hpp from kfs_grpc_service.hpp  and added there only forward declaration. It turned out that we had indirect include dependency here